Services/Administration/OJS.Services.Administration.Business/Contests/ContestsBusinessService.cs

```diff
@@ -370,6 +378,22 @@ private async Task AddIpsToContest(Contest contest, string? mergedIps)
         }
     }
 
+    private void ClearContestCache(
+        Contest contest,
+        ContestAdministrationModel model,
+        bool originalIsVisibleState,
+        string? originalContestPassword,
+        string? originalPracticePassword)
+    {
+        if (contest.IsDeleted ||
+            originalIsVisibleState != model.IsVisible ||
+            originalContestPassword != model.ContestPassword ||
+            originalPracticePassword != model.PracticePassword)
+        {
+            this.contestParticipantsCacheService.ClearContestCacheByContestId(contest.Id);
+        }
```

Comment:
Why check all this data? Isn't it more robust to just clear the cache when anything on the contest is updated?

Reply:
Done.